addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -425,5 +425,10 @@ <H2>X-Debug-Token Information Leak</H2>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XDebugTokenScanRule.java">XDebugTokenScanRule.java</a>
 
+<H2>Same Origin Method Execution</H2>
+TThis scanning rule examines JSONP endpoints to identify user-controllable function names.
+<p>
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/SameOriginMethodExecutionScanRule.java">SameOriginMethodExecutionScanRule.java</a>
```

Comment:
I should have noticed this earlier, but brand new rules are to be added in the alpha add-on (pscanrulesAlpha).

Reply:
Updated to alpha quality rule :white_check_mark: